icu4c/source/i18n/currunit.cpp

```diff
@@ -20,7 +20,7 @@
 #include "uinvchar.h"
 #include "charstr.h"
 
-static constexpr char16_t kDefaultCurrency[] = u"XXX";
+static constexpr char16_t kDefaultCurrency[] = {u'X', u'X', u'X', u'\0'};
```

Comment:
Can you change these to `char16_t*` type?

Reply:
clang yelled at me in the previous push
```
cannot initialize a type of char16_t const with an lvalue of const char16_t*
```

Reply:
How about

```cpp
static constexpr const char16_t* kDefaultCurrency = u"XXX";
```

?  It might take some experimentation to get the compilers happy, but the `{}` syntax for strings is a style we have been moving away from and is not unlikely to get hastily changed again the next time someone works on this file.

Reply:
ah i see. should be fixed in this push. (Not sure why it was writable to begin with, being a `static constexpr` field)